gax/src/streamingCalls/streaming.ts

```diff
@@ -158,21 +352,168 @@ export class StreamProxy extends duplexify implements GRPCCallResult {
     });
   }
 
+  defaultShouldRetry(error: GoogleError, retry: RetryOptions) {
+    if (
+      Array.isArray(retry.retryCodesOrShouldRetryFn) &&
+      retry.retryCodesOrShouldRetryFn.indexOf(error!.code!) < 0
+    ) {
+      return false;
+    }
+    return true;
+  }
+
+  /**
+   * Forward events from an API request stream to the user's stream.
+   * @param {Stream} stream - The API request stream.
+   * @param {RetryOptions} retry - Configures the exceptions upon which the
+   *   function eshould retry, and the parameters to the exponential backoff retry
+   *   algorithm.
+   */
+  forwardEventsNewImplementation(
```

Comment:
`NewImplementation` does not age well :)

Reply:
lol, agreed, I've renamed